Bio/pairwise2.py

```diff
@@ -960,6 +1005,8 @@ def _clean_alignments(alignments):
     Remove duplicates, make sure begin and end are set correctly, remove
     empty alignments.
     """
+    Alignment = namedtuple("Alignment",
+                           ("seqA, seqB, score, start, end"))
```

Comment:
Fits as one line of 74 characters - but can leave this until we run black on the whole file later.

Reply:
OK.